sr/robot3/kch.py

```diff
@@ -25,8 +25,8 @@
 
 
 @unique
-class RobotLEDs(IntEnum):
-    """Mapping of LEDs to GPIO Pins."""
+class RobotLEDPin(IntEnum):
+    """Pin for a corresponding LED channel"""
```

Comment:
Not a pin
```suggestion
class RobotLedGpio(IntEnum):
    """GPIO for a corresponding LED channel"""
```

Reply:
It's the GPIO pin number, right? That feels accurate enough

Reply:
![](https://i.kym-cdn.com/entries/icons/facebook/000/019/304/Old_Man_Yells_at_cloud_cover.jpg)